bin/cron/applab_datasets/spotify

```diff
@@ -63,6 +71,19 @@ def get_viral50_data(url)
   return records, columns
 end
 
+# Only check artist and song names for profanity
+def profanity_check(line)
```

Comment:
nit: Can this be renamed? From the name `profanity_check` I thought this was going to actually *do* the check. maybe `get_string_to_check` or something like that?